lldb/source/Commands/CommandObjectStats.cpp

```diff
@@ -83,13 +86,17 @@ class CommandObjectStatsDump : public CommandObjectParsed {
 
     void OptionParsingStarting(ExecutionContext *execution_context) override {
       m_all_targets = false;
+      m_stats_options = StatisticsOptions();
     }
 
     llvm::ArrayRef<OptionDefinition> GetDefinitions() override {
       return llvm::ArrayRef(g_statistics_dump_options);
     }
 
+    StatisticsOptions GetStatisticsOptions() { return m_stats_options; }
```

Comment:
return a `const StatisticsOptions&` if you want to. The struct isn't big now, so no big deal.